libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -1443,6 +1444,7 @@ open class SalesforceSDKManager protected constructor(
          * TODO: Remove the suppress lint annotation once the Android context is no longer retained.
          */
         @JvmField
+        @SuppressLint("StaticFieldLeak")
```

Comment:
I'd thought this suppress was in place quite some time ago, since there's a TODO already around the bigger issue present here.